content/home.html

```diff
@@ -110,7 +110,7 @@ <h3><a href="https://sourceforge.net/p/pdl/news/">Recent News</a></h3>
 <script type="text/javascript">
 	// Start slideshow plugin.
 	$('.slideshow').cycle({
-		fx: 'fade,curtainX,zoom,turnDown,scrollRight',
+        fx: 'fade,curtainX,turnDown,scrollRight',
```

Comment:
Wouldn't have mentioned only this, but there's the other point below. Reindenting this seems like not an improvement?